script/Makefile

```diff
@@ -305,10 +305,10 @@ dep:
 	go mod tidy
 
 lint:
-	GOGC=$(LINT_GOGC) golangci-lint run --out-format tab --deadline $(LINT_DEADLINE)
+	GOGC=$(LINT_GOGC) golangci-lint run --config .golangci.yml --out-format tab --deadline $(LINT_DEADLINE) --verbose
```

Comment:
There's nothing wrong with this but do we need this?  It looks like the default always looks for `.golangci.yml` in the current dir:
https://golangci-lint.run/usage/configuration/

Reply:
Default lookup is not working for me (I am on MacOS)